tests/installation/ipxe_install.pm

```diff
@@ -46,12 +46,14 @@ sub poweron_host {
 }
 
 sub set_pxe_boot {
+    my $is_uefi = get_var('IPXE_UEFI');
     while (1) {
         my $stdout = ipmitool('chassis bootparam get 5');
-        last if $stdout =~ m/Force PXE/;
+        last if $stdout =~ is_uefi ? "m/(Force PXE|BIOS EFI boot)/" : "m/Force PXE/";
         diag "setting boot device to pxe";
-        my $options = get_var('IPXE_UEFI') ? 'options=efiboot' : '';
+        my $options = $is_uefi ? 'options=efiboot' : '';
         ipmitool("chassis bootdev pxe ${options}");
+        ipmitool("raw 0x00 0x08 0x05 0xe0 0x04 0x00 0x00 0x00") if $is_uefi;
```

Comment:
would it be useful to add a comment explaining line 56?

Reply:
While I like magic in code, you're right. I added a comment pointing to the section in the IPMI specification